pkgs/applications/science/math/R/default.nix

```diff
@@ -30,19 +30,6 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./no-usr-local-search-paths.patch
-  ] ++ lib.optionals (!withRecommendedPackages) [
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done

Reply:
I think you didn't push this.

Reply:
Sorry, I'm not sure what you mean. I can only see two calls two fetchpatch, both of them are removed by d37edcd, and d37edcd was pushed 2 days ago. You did make your original comment before I pushed it. Is this resolved now?

Reply:
Sandro means this https://github.com/PhDyellow/nixpkgs/blob/d37edcd3d5d764b2ac2b9b03835adf744dc596c9/pkgs/applications/science/math/R/default.nix#L4

Reply:
Ah, thanks @Artturin!

Reply:
Latest commit removes fetchpatch from function inputs.

Reply:
Thanks for your feedback @SuperSandro2000